openmetadata-airflow-apis/openmetadata_managed_apis/workflows/ingestion/common.py

```diff
@@ -297,6 +298,8 @@ def build_dag(
             retries=0,
             # each DAG will call its own OpenMetadataWorkflowConfig
             on_failure_callback=partial(send_failed_status_callback, workflow_config),
+            # Add tag and ownership to easily identify DAGs generated by OM
+            owner="openmetadata",
```

Comment:
why not pass the owner from the user who is deployed it. We already recording this in IngestionPipeline Owner